docker/run.sh

```diff
@@ -137,11 +137,13 @@ function run_tests {
         argv_str=$(printf ' %q' "$TEST" "$@")
         su cchq -c "/bin/bash ../run_tests $argv_str" 2>&1
         log_group_end  # only log group end on success (notice: `set -e`)
-        [ "$TEST" == "python-sharded-and-javascript" ] && scripts/test-prod-entrypoints.sh
-        [ "$TEST" == "python-sharded-and-javascript" ] && scripts/test-make-requirements.sh
-        [ "$TEST" == "python-sharded-and-javascript" ] && scripts/test-serializer-pickle-files.sh
-        [ "$TEST" == "python-sharded-and-javascript" -o "$TEST_MIGRATIONS" ] && scripts/test-django-migrations.sh
-        [ "$TEST" == "python-sharded-and-javascript" ] && scripts/track-dependency-status.sh
+        if [ "$TEST" == "python-sharded-and-javascript" ]; then
+            su cchq -c scripts/test-prod-entrypoints.sh
+            scripts/test-make-requirements.sh
+            scripts/test-serializer-pickle-files.sh
+            [ "$TEST_MIGRATIONS" ] && su cchq -c scripts/test-django-migrations.sh
```

Comment:
I notice now that this changed from _OR_ to _AND_. After a quick search I don't see where `TEST_MIGRATIONS` is used, so I'm inclined to leave it as is. Maybe I'll remove it in a later PR since it appears to be unused.

Reply:
As it stands now it won't ever run the script right? Whereas before it would have run if `"$TEST" == "python-sharded-and-javascript"`.

Reply:
Yes, of course. Good catch. https://github.com/dimagi/commcare-hq/pull/32590